geode-core/src/main/java/org/apache/geode/distributed/internal/InternalLocator.java

```diff
@@ -478,9 +478,8 @@ private InternalLocator(int port, File logF, File stateF, InternalLogWriter logW
     if (logWriter == null) {
       logWriter = LogWriterFactory.createLogWriterLogger(false, false, this.config,
           !startDistributedSystem);
-      if (logger.isDebugEnabled()) {
+      if (logger.isDebugEnabled())
```

Comment:
We've always had {'s in our style guidelines. So most of us are adding them in for small if-blocks rather than removing them.